### DISCUSSION TITLE
Tech Demo Store

### DISCUSSION THREAD

**Discussion by uandiweb** [scandipwa/scandipwa#2395] [General]
Hi to all,

Is there an option to install the sample demo data you have in the "Tech" demo store: https://tech-demo.scandipwa.com/

Thank you

**Comment by lianastaskevica** [comment]
Hello @uandiweb! The demo data from the tech-demo store is not publicly available. 

If you need to have a demo data to work with, you can require our scandipwa/sample-data package or use Magento sample data:)
